### DISCUSSION TITLE
Create notification that does not disappear until manually dismissed

### DISCUSSION THREAD

**Discussion by dudefoxlive** [Windos/BurntToast#214] [Q&A]
I am looking to create a notification that contains info about Terms of Use. I was hoping to make it not disappear until a user manually clicks Dismiss. Is this possible with Burntoast?

**Comment by jaredn13** [comment]
Not an answer but I am also looking for this functionality.  While also removing the X to close the box, in addition to having the window persistent until one of the presented buttons is pressed as described above.

**Comment by oquendoG** [comment]
I need this too, please help

**Comment by unalignedcoder** [comment]
I'd be happy with a way to set the duration of the notification. Is there one?

**Comment by Windos** [comment]
At its core, this module is just using the Windows API for toast notifications so some of the limitations are those imposed by Microsoft and isn't something that can't be worked around.

The best you can do is tweak the "scenario".

I think this is the longest I've managed to get a toast to display: https://github.com/Windos/Talks/blob/main/2021/02%20-%20February/23%20-%20DuPSUG%20-%20Croque%20Monsieur/Demos/03_Scenario.ps1

It did require selecting a looping sound and it will display for a very long time, but there's no guarantee it'll stay until the user has seen it.

**Comment by ITistheworst** [comment]
[I also posted this on #97](https://github.com/Windos/BurntToast/discussions/97#discussioncomment-12311041), which seems to be a duplicate:

I have found that `-Scenario Reminder` produces a notification that stays on screen until the user interacts with it. I have been using with `-Duration Long`, but I am not sure if that is necessary. E.g:

```powershell
$content = New-BTContent -Visual $visual -Duration Long -Scenario Reminder
```

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
